terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/exceptionhandling/ExceptionHandlingTest.java

```diff
@@ -46,8 +45,7 @@ public ExceptionHandlingTest() {
     public void test01_01_requestControllerHandling() {
         driver.findElement(By.id("requestControllerHandling_01_01")).click();
 
-        // WARN Level Log

```

Comment:
line comments is deleted...
However I think that comments are unnecessary, too.